docs/clients.md

```diff
@@ -55,7 +55,7 @@ To use these, add the appropriate module to your build, and then pass an instanc
 For Akka HTTP, we use `AkkaHttpClient`:
 
 ```scala
-val client = ElasticClient(AkkaHttpClient(AkkaHttpClientSettings(List("http://host1:9200"))))
```

Comment:
in my tests I got an error "invalid input" when the scheme was added here. the https config settings should be used to define the scheme